CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.1.3
+-------------------
+Bugfixes:
+* Removed upper bound on `cattrs` version.
+* Objects unknown types or with nested objects of unknown types (e.g. features not yet fully supported in the API) can be safely structured now.
```

Comment:
I think we need to describe it more precisely. Like:
While structuring objects with unknown `spec_value` corresponding classes are generated in runtime now. Thus objects like filters, rule actions and others that are not fully documented in api/experimental/removed in past from toloka-kit could be safely structured now.